pandas/tests/indexes/period/test_arithmetic.py

```diff
@@ -613,42 +613,42 @@ def test_pi_ops(self):
         exp = pd.Index([0, -1, -2, -3], name='idx')
         tm.assert_index_equal(result, exp)
 
-    def test_pi_ops_errors(self):
+    @pytest.mark.parametrize('ng', ["str", 1.5])
+    def test_pi_ops_errors(self, ng):
         idx = PeriodIndex(['2011-01', '2011-02', '2011-03', '2011-04'],
                           freq='M', name='idx')
         ser = pd.Series(idx)
 
         msg = r"unsupported operand type\(s\)"
 
         for obj in [idx, ser]:
```

Comment:
still need this loop here?

Reply:
There are a bunch of borderline cases like this left for the next pass.

Reply:
thats fine